stable/pipelines/values.yaml

```diff
@@ -513,6 +513,9 @@ pipelines:
       ignoreTLSErrors: false
       controlplaneVersion: "{{ default .Chart.AppVersion .Values.pipelines.pipelinesInit.image.tag }}"
       buildplaneVersion: "{{ default .Chart.AppVersion .Values.pipelines.pipelinesInit.image.tag }}"
+      accessControlAllowOrigins:
+        - {{ .Values.pipelines.accessControlAllowOrigins_0 }}
```

Comment:
Please add this to the values. Also, why not just use the list as is?

Reply:
@danielezer added to the values. The reason for not using list was values for saas are coming like this from saas-deployer.